src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -308,6 +285,32 @@ public TimeSpan Expect100ContinueTimeout
             }
         }
 
+        /// <summary>
+        /// Defines the initial HTTP2 stream receive window size for all connections opened by the this <see cref="SocketsHttpHandler"/>.
+        /// </summary>
+        /// <remarks>
+        /// Larger the values may lead to faster download speed, but potentially higher memory footprint.
+        /// The property must be set to a value 65535 and the configured maximum window size, which is 16777216 by default.
```

Comment:
```suggestion
        /// The property must be set to a value between 65535 and the configured maximum window size, which is 16777216 by default.
```
Is this what you intended?